scripts/common/get_rest_api.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
```

Comment:
not sure what is this for.

Reply:
It contains a shell script routine which returns the REST API server URL, asking  the FogLAMP Core